package.json

```diff
@@ -9,6 +9,7 @@
   "main": "webpack.config.js",
   "scripts": {
     "hot": "webpack-dev-server --inline --hot --history-api-fallback --open",
+    "hot:docker": "webpack-dev-server --host 0.0.0.0 --inline --hot --history-api-fallback --open",
```

Comment:
What is the difference between "hot" and "hot:docker" except for changing the host address? Why is there the change in the address?

Reply:
The change in address was required in order to make the server visible from the Docker container. Without it, you'd just see an empty page at localhost:8080.

Reply:
I think the expectations for "hot:docker" would be that it should run the docker version of the app. But running this myself doesn't run the app in docker. It just uses webpack for me.

So two things I would like to change/update:
- Instructions/explanation for how docker and webpack will need to work together so folks who are interested can learn about docker.
- Make the ``hot:docker`` script run everything we need to without needing to run anything else besides having the docker desktop app open in the background. So if we need to add extra commands within in the ``hot:docker`` script, or if we want to create a script file and run that script file inside of the ``hot:docker`` command, let's do that.